packages/salesforcedx-vscode-core/src/messages/i18n.ts

```diff
@@ -243,5 +243,7 @@ export const messages = {
     'No default org is set. Run "SFDX: Create a Default Scratch Org" or "SFDX: Authorize an Org" to set one.',
   error_no_default_devhubusername:
     'No default Dev Hub is set. Run "SFDX: Authorize a Dev Hub" to set one.',
-  custom_output_directory: 'Choose a Custom Directory'
+  custom_output_directory: 'Choose a Custom Directory',
+  warning_using_global_username:
```

Comment:
@ruthemmanuelle this was the warning message you'd looked at this morning when I was writing the tests.